vcpkg/users/port-overlays.md

```diff
@@ -0,0 +1,80 @@
+---
+title: Port overlays
+description: This article describes overlay ports in vcpkg. Overlay ports are used to force a specific port definition to be used by vcpkg during the package installation process.
+author: data-queue
+ms.author: danshaw2
+ms.date: 9/20/2023
+ms.prod: vcpkg
+ms.topic: conceptual
+---
+
+# Overlay ports
+
+## Overview
+An overlay port can act as a drop-in replacement for an existing port. An overlay port can also add a new port that is otherwise not available in a [registry](../maintainers/registries.md). While [resolving package names](registries.md#package-name-resolution), overlay ports take priority, forcing overlay ports to be selected.
+
+Naming conflicts can arise when multiple overlays introduce the same port. For instance, overlay directories `[dirA, dirB]` may both introduce a port with the same name: `dirA/zlib` and `dirB/zlib`. vcpkg resolves `zlib` in order of which directory is specified first. In this example, `dirA/zlib` is selected as the `zlib` port. If overlay port options are specified in multiple places, overlay resolution priority follows first from the command line, the manifest, and finally the environment.
+
+## Configuration Syntax
+
+A directory refers to a set of overlay ports and can be specified as follows: 
+- Single overlay port: `<directory>/sqlite3` refers to a single port
+- Directory of overlay ports: `<directory>` refers to a directory of ports
+A valid port must contain both `vcpkg.json` and `portfile.cmake`.
+
+You can add an overlay port in several ways:
+* Command-line: Add one or multiple `--overlay-ports=<directory>` options to your vcpkg command
+* [Manifest](../reference/vcpkg-configuration-json.md#overlay-ports): Populate the `"overlay-ports"` array in `vcpkg-configuration.json`
+* [Environmental variable](./config-environment.md#vcpkg_overlay_ports): Set `VCPKG_OVERLAY_PORTS` to a list of directory paths
+
+### Example: Overlay Ports Example
+
+Given this directory structure:
+```
+team-ports/
+|-- sqlite3/
+|---- vcpkg.json
+|---- portfile.cmake
+|-- rapidjson/
+|---- vcpkg.json
+|---- portfile.cmake
+|-- curl/
+|---- vcpkg.json
+|---- portfile.cmake
+
+my-ports/
+|-- sqlite3/
+|---- vcpkg.json
+|---- portfile.cmake
+|-- rapidjson/
+|---- vcpkg.json
+|---- portfile.cmake
+
+vcpkg
+|-- ports/
+|---- <upstream ports>
+|-- vcpkg.exe
+```
+
+Running:
+```
+vcpkg.exe install sqlite3 --overlay-ports=my-ports --overlay-ports=team-ports
+```
```

Comment:
Set the code-block language to PowerShell or Bash, since this is a conceptual article, we don't need to provide instructions for each platform, we just pick one. Also, remove the `.exe` from the command.